pkgs/tools/networking/ivpn/default.nix

```diff
@@ -31,16 +40,58 @@ builtins.mapAttrs (pname: attrs: buildGoModule (attrs // rec {
     homepage = "https://www.ivpn.net/apps";
     changelog = "https://github.com/ivpn/desktop-app/releases/tag/v${version}";
     license = licenses.gpl3Only;
-    maintainers = with maintainers; [ urandom ];
+    maintainers = with maintainers; [ urandom ataraxiasjel ];
   };
 })) {
   ivpn = {
     modRoot = "cli";
-    vendorHash = "sha256-5FvKR1Kz91Yi/uILVFyJRnwFZSmZ5qnotXqOI4fKLbY=";
+    vendorHash = "sha256-T49AE3SUmdP3Tu9Sp5C/QryKDto/NzEqRuUQ3+aJFL0=";
   };
   ivpn-service = {
     modRoot = "daemon";
     vendorHash = "sha256-9Rk6ruMpyWtQe+90kw4F8OLq7/JcDSrG6ufkfcrS4W8=";
-    buildInputs = [ wirelesstools ];
+    nativeBuildInputs = [ makeWrapper ];
+    buildInputs = [
+      dnscrypt-proxy2
+      iproute2
+      obfs4
+      openvpn
+      wireguard-tools
+      wirelesstools
```

Comment:
Are those even used somewhere? buildInputs is not affecting the usage of wrapProgram below.

Reply:
It was needed before, but with the new patch (hardcoded paths to binaries), it is no longer necessary.